karma.conf.js

```diff
@@ -39,6 +39,7 @@ module.exports = function(config) {
     files: [
       'src/**/*.ts'
     ],
+    exclude: ["src/payment-providers/declarations.d.ts"],
```

Comment:
Where is the other place this is defined?

Reply:
There isn't another place it gets called. Testing works with it excluded.

For tests, Karma-typescript transpiles `.d.ts` files incorrectly, which results in failed tests as the Typescript compiler resolves them as ambient modules. Normally devs put their declaration files under a `@types` folder and exclude that folder from tests when using karma-typescript.